docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.9"
+services:
+  app:
+    command: /bin/bash
```

Comment:
Blocking: Don't define `command:` as part of docker-compose.yml. Define the default command as part of Dockerfile, and if you want to run an exceptional command as part of development, put that in `docker-compose.override.yml`